packages/node_modules/@ciscospark/redux-module-media/src/reducer.js

```diff
@@ -23,8 +23,8 @@ export const initialState = fromJS({
     call: null
   },
   callState: {
-    sendingAudio: true,
-    sendingVideo: true
+    sendingAudio: false,
+    sendingVideo: false
```

Comment:
We're actually looking for ```media.getIn([`callState`, `localMediaStream`]);``` when determining if the call controls display. `sendingAudio` and `sendingVideo` just control how the button is styled.

Just gave it a shot with 2G network throttling and it worked fine.